ch03tests/04EnergyExample.ipynb

```diff
@@ -415,7 +415,7 @@
    "source": [
     "%%bash\n",
     "cd diffusion\n",
-    "pytest --cov"
+    "pytest --cov=\"diffusion\""
```

Comment:
Are the quotes in case of directories with spaces? Is it simpler to leave them out?